### DISCUSSION TITLE
Github and jenkins

### DISCUSSION THREAD

**Discussion by aryejfa** [community/community#18906] [Other Feature Feedback, Questions, & Ideas] [answered]
How to unify github with jenkins

**Comment by mAlex28** [comment]
I found this article online: https://www.blazemeter.com/blog/how-to-integrate-your-github-repository-to-your-jenkins-project
Hope it helps

**Reply by aryejfa** [reply]
Thank you frend

**Reply by aryejfa** [reply]
Thank you frend

**Comment by melvincwng** [comment, marked as answer]
Hi @aryejfa, for linking of GitHub with Jenkins, kindly visit the following youtube tutorial & article below:
1. [Jenkins GitHub Integration - by Cameron McKenzie](https://www.youtube.com/watch?v=rwbN6JjDCXo)
2. [How to Integrate Your GitHub Repository to Your Jenkins Project](https://www.blazemeter.com/blog/how-to-integrate-your-github-repository-to-your-jenkins-project)

Hope this helps!

**Reply by aryejfa** [reply]
Thank you frend
